test/test_other.py

```diff
@@ -9182,6 +9182,20 @@ def test_asyncify_advise(self):
     self.assertContained('[asyncify] g can', out)
     self.assertContained('[asyncify] i can', out)
 
+  def test_asyncify_stack_overflow(self):
+    args = ['-sASYNCIFY', '-sASYNCIFY_STACK_SIZE=4']
```

Comment:
`self.emcc_args += `  (these will be added automatically to the `emcc_args` specified via the parameter below.